content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,255 @@
+===================
+Track and bill time
+===================
+
+*Helpdesk* provides teams with the ability to track the amount of hours spent working on a ticket,
+and to bill a customer for that time. Through integrations with the *Sales*, *Timesheets*, *Project*
+and *Accounting* applications, customers can be charged once the work is completed, or before it has
+even begun.
+
+.. warning::
+   Since the *Track & Bill Time* features require integration with other applications, enabling them
+   may result in the installation of additional modules or applications.
+
+   Installing a new application on a *One-App-Free* database triggers a 15-day trial. At the end of
+   the trial, if a paid subscription has not been added to the database, it will no longer be active
+   or accessible.
+
+Configure track and bill time features
+======================================
+
+Before a customer can be invoiced for support services, the *Track & Bill Time* features **must** be
+enabled on each *Helpdesk* team individually.
+
+Enable track and bill time on a team
+------------------------------------
+
+To view and enable the *Track & Bill Time* features on a *Helpdesk* team, first navigate to
+:menuselection:`Helpdesk app --> Configuration --> Helpdesk Teams`. Then, select a team from the
+list, or create a :doc:`new one <../overview/getting_started>`. This reveals a team's settings page.
+
+On the team's settings page, click :guilabel:`Edit`, then scroll to the :guilabel:`Track & Bill
+Time` section. Tick the boxes labeled :guilabel:`Timesheets` and :guilabel:`Time Billing`.
+
+Once the :guilabel:`Timesheets` box is checked, a new field appears, labeled :guilabel:`Project`.
+
+.. note::
+   If this is the first time this feature has been enabled on this database, the page may need to be
+   manually saved and refreshed before the :guilabel:`Project` field appears.
+
+The project selected in this field represents where all the timesheets for this team's tickets are
+recorded. Click into the :guilabel:`Project` drop-down menu to select a project.
+
+To create a new project where the timesheets are recorded, click into the :guilabel:`Project`
+drop-down menu, type a name for the project, and then click :guilabel:`Create` from the drop-down
+menu beneath.
+
+.. image:: track_and_bill/track-bill-enable-settings.png
+   :align: center
+   :alt: The settings page of a Helpdesk team with the track and bill time settings enabled.
+
+.. _helpdesk/configure-service-products:
+
+Configure service products
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When the :guilabel:`Time Billing` feature is enabled, a new product is created in the *Sales* app
+called *Service on Timesheets*. This product can be found under :menuselection:`Sales app-->
+Products --> Products`. Then, search for `Service on Timesheets` in the :guilabel:`Search...` bar.
+This is the product that is used when invoicing for *post-paid support services* **after** they have
+been completed.
```

Comment:
Is 'post-paid' a term for Odoo, or is it called 'after-market' meaning after purchasing? Just curious- not a necessary change.

Reply:
I don't know for sure if it is just an Odoo term, but I wanted to use the terms from the previous docs I am basing this one on